app/models/user.rb

```diff
@@ -1,5 +1,5 @@
 class User < ApplicationRecord
-  belongs_to :organisation
+  belongs_to :organisation, optional: true
```

Comment:
Also, please add these changes (optional belongs_to, removed validations) to the user specs as well.